src/components/Issues.js

```diff
@@ -71,6 +78,12 @@ function Issues({repoName, owner}) {
       <CardHeader>
         <h1>Issues</h1>
       </CardHeader>
+      <Card style={{paddingBottom: 0, paddingTop: "0.5em"}}>
+        <Flex>
+          <Paragraph>Issues can be filtered by to show good opportunities to assist with this repository.</Paragraph>
```

Comment:
"can be filtered to show" should be used

Reply:
Good catch! I think I was originally going to say something about "filtered by labels". I'll probably change the wording a little more to reflect the fact that the labels are indeed part of the filtering strategy.